src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -75,15 +75,25 @@ class TosService (val directoryDao: DirectoryDAO, val registrationDao: Registrat
     *         IO[Some(false)] if ToS is enabled and the user hasn't accepted
     *         IO[None] if ToS is disabled
     */
-  def getTosStatus(user: WorkbenchSubject, samRequestContext: SamRequestContext): IO[Option[Boolean]] = {
+  def getTosStatus(userId: WorkbenchUserId, samRequestContext: SamRequestContext): IO[Option[Boolean]] = {
```

Comment:
This was preexisting so don't need to change anything, but the fact that these return `Option[Boolean]` is a bit strange to me. Eventually ToS will be enabled everywhere so will we then rip out the `Option`?

Generally I prefer clean types in the service layer and have the call site deal with whether a thing is enabled or not.

Reply:
I don't disagree. I think one use case may be the Verily deployment where ToS may never be used but I don't know.